puppeteer/src/domains/automation/services/puppeteerService.ts

```
@@ -189,34 +205,34 @@ export class PuppeteerService {
       throw new Error('Browser not initialized. Call initialize() first.');
     }
 
     try {
-      const scrollDistance = distance || (direction === 'down' ? 
+      const scrollDistance = distance ?? (direction === 'down' ?
         'window.innerHeight' : '-window.innerHeight');
-      
-      await this.page.evaluate((dist) => {
+
+      await this.page.evaluate((dist: string | number) => {
         window.scrollBy(0, typeof dist === 'string' ? eval(dist) : dist);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)